Tests/IgnoredFunctions.hh

```diff
@@ -0,0 +1,11 @@
+// RUN: %idt -export-macro IDT_TEST_ABI -ignore foo,bar %s 2>&1 | %FileCheck %s
+// RUN: %idt -export-macro IDT_TEST_ABI -ignore foo -ignore bar %s 2>&1 | %FileCheck %s
```

Comment:
Minor nit: `IDT_TEST_ABI` is not referenced, so I think that it would be safe to remove the `-export-macro IDT_TEST_ABI` option from the invocation.

Reply:
That's what I get for copy-and-pasting. I'll remove it.

Reply:
Haha, guess that is needed after all!